react/global.css

```diff
@@ -133,10 +133,14 @@ body.vtex-filter-popup-open {
   }
 }
 
-@media only screen and (min-width: calc(40em + 1px)) {
+@media only screen and (min-width: 40em) {
   .vtex-search-result__border {
     display: none;
   }
+
+  .vtex-search__term {
```

Comment:
With tachyons you can do `dn-ns`

Reply:
Because this is inside a breakpoint, so I can't make it ```display: none``` only with the class ```dn-ns``` from tachyons